core/tracepoint/enable_spec.rb

```diff
@@ -0,0 +1,20 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#enable' do
+    context 'activates the trace' do
```

Comment:
This is not really context. Moreover, the specs do not show the effect of calling `enable` other than changing the result of successive calls. We need a test that shows that a trace doesn't call the block until it's enabled...

Reply:
okay, I have updated the PR.